File: 프로그래머스/unrated/181844. 배열의 원소 삭제하기/배열의 원소 삭제하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<int> delete_list) {
    vector<int> answer;
    for (int i = 0; i < arr.size(); i++)
    {
        int sign = 0;
        for (int j = 0; j < delete_list.size(); j++)
        {
            if (arr[i] == delete_list[j])
                sign = 1;
        }
        if (sign == 0)
            answer.push_back(arr[i]);
    }
    
    return answer;
}

File: 프로그래머스/0/181844. 배열의 원소 삭제하기/배열의 원소 삭제하기.java
import java.util.Arrays;
import java.util.stream.IntStream;

class Solution {
    public int[] solution(int[] arr, int[] delete_list) {
        return IntStream.of(arr)
                .filter(i -> Arrays.stream(delete_list).noneMatch(j -> j == i))
                .toArray();
    }
}